### DISCUSSION TITLE
Svelte render plugin for v2

### DISCUSSION THREAD

**Discussion by Ni55aN** [retejs/rete#613] [Polls]
I'm considering creating a Svelte render plugin and would like to hear your thoughts on whether it's needed for you. I'd appreciate it if you could take a moment to share your opinion by leaving a comment.

Thank you for your participation!

**Comment by saalaus** [comment]
I am developing an application using rete and svelte(unofficial render plugin).
It would be very cool to see an official svelte render plugin

**Reply by ShaitanLyss** [reply]
I too plan to use rete + svelte, svelte is an enjoyable framework to use and in my experience it's faster to develop with it. I think it would fit very well with how you designed rete.

**Comment by FilterUnfiltered** [comment]
I primarily use svelte and a potential project of mine could really use something like this :)

**Comment by Ni55aN** [comment]
@FilterUnfiltered @ShaitanLyss @saalaus Svelte plugin just [has been released](https://twitter.com/rete_js/status/1675577935233703939)
